data/mods/Aftershock/items/gun/7.50mm.json

```
@@ -0,0 +1,101 @@
+[
+  {
+    "id": "afs_landfall_gun",
+    "type": "GUN",
+    "copy-from": "rifle_22",
+    "name": { "str": "landfall survival gun" },
+    "description": "A lightweight, bolt-action gun that is barely more than a 6.55mm barrel mounted within a flimsy metal assembly.  Cheaply made and outdated in all respects, it's only useful in the most extenuating of circumstances.",
+    "price": 5000,
+    "price_postapoc": 5000,
+    "material": [ "steel", "plastic" ],
+    "flags": [ "NEVER_JAMS", "RELOAD_EJECT" ],
+    "dispersion": 250,
+    "longest_side": "60 cm",
+    "reload": 100,
+    "durability": 5,
+    "blackpowder_tolerance": 60,
+    "ammo": [ "afs_7.50mm" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "afs_7.50mm": 1 } } ],
+    "faults": [ "fault_gun_blackpowder", "fault_gun_dirt" ]
+  },
+  {
+    "id": "afs_Accipitermg",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "Accipiter Hawk-00" },
+    "description": "A machine gun manufactured by Accipiter Firearms and distributed by Wraitheon.  It is heavy, inaccurate, and difficult to control, the Hawk-00 was designed to fire many rounds very quickly.  It is more ideal to use when mounted.",
+    "weight": "7500 g",
+    "volume": "10610 ml",
+    "longest_side": "1035 mm",
+    "price": 750000,
+    "price_postapoc": 8000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "superalloy", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "afs_7.50mm" ],
+    "dispersion": 200,
+    "durability": 7,
+    "min_cycle_recoil": 1350,
+    "reload": 400,
+    "ranged_damage": { "damage_type": "bullet", "amount": -14 },
+    "modes": [ [ "DEFAULT", "burst", 4 ] ],
+    "built_in_mods": [ "bipod" ],
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ],
+      [ "stock", 1 ]
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "item_restriction": [ "afs_UICASTA30", "afs_UICASTA100drum" ]
+      }
+    ]
+  },
+  {
+    "id": "afs_gibrifle",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "G&W SR-P9" },
+    "description": "A classic bullpupped service rifle manufactured for UICA use as a joint venture between Wraitheon and Gibson Armory.  While it is less commonly used by UICA today, many a private military and contractor has found a use for the SR-P9 to keep it on the market long past its initial production date.",
```

```suggestion
    "description": "An eminently sturdy and reliable  bullpop rifle iused by UICA's military during early reclamation expeditions.  Although its been out of service for nearly two decades, many a private military and contractor has found a use for the SR-P9, and it remains common long past its initial production date.",
```
There was something that sounded wrong about the last part of the last sentence, but Im not necesarily sure that my suggestion for it is much better.

---

I think maybe the "production date" is the issue here, so I'll probably change that to "initial introduction into service"